client/src/Main.elm

```diff
@@ -1,6 +1,6 @@
-module Main exposing (..)
+module Main exposing (main, subscriptions)
 
-import Html exposing (program)
+import Browser exposing (element)
```

Comment:
`Html.program` has changed to `Browser.*`. There are a couple options here, but `Browser.element` seems to be closest to how it was set up before. It looks for an HTML node and embeds the Elm application onto it.